### ISSUE TITLE
[Bug] Github Action Excavator auto update is not using the right version from checkver

### ISSUE THREAD

**Issue by LazyGeniusMan** [ScoopInstaller/Scoop#4617]
## Bug Report
First, is it possible to enable Issue for ScoopInstaller/GithubActions and move this issue there ? I believe this should be there.

#### Current Behavior

Detected version from checkver is not used in auto update

#### Expected Behavior

Detected version from checkver is used in auto update

#### Additional context/output

For example, `mindforger` detected checkver is 1.53.0[^1] but in auto update the version is 1.1.1[^2], this also happen in other excavator run[^3].

It's also happen in other manifest, for example: `ldplayer`[^4], `kate`[^5], `mqttx`[^6] and many others.

Is this bug? I tried to locally run checkver for `mqttx` and it's updated correctly.

#### Possible Solution

I haven't found yet what's wrong.

[^1]: https://github.com/ScoopInstaller/Extras/runs/4681872557?check_suite_focus=true#step:3:249
[^2]: https://github.com/ScoopInstaller/Extras/runs/4681872557?check_suite_focus=true#step:3:251
[^3]: https://github.com/ScoopInstaller/Extras/runs/4682667936?check_suite_focus=true#step:3:267
[^4]: https://github.com/ScoopInstaller/Extras/runs/4681872557?check_suite_focus=true#step:3:257
[^5]: https://github.com/ScoopInstaller/Extras/runs/4681872557?check_suite_focus=true#step:3:263
[^6]: https://github.com/ScoopInstaller/Extras/runs/4681872557?check_suite_focus=true#step:3:273

**Comment by rashil2000** [comment]
Yes, we have noticed this too. From my observation, nothing is wrong with either auto-pr.ps1 or checkver.ps1. All version fetching logic is inside checkver.ps1, but it seems to be running correctly locally. Also, Main and Versions do not have this issue (yet).

> First, is it possible to enable Issue for ScoopInstaller/GithubActions and move this issue there ? I believe this should be there.

Yes. @issaclin32 can you enable issues for the Actions repo?

**Comment by LazyGeniusMan** [comment]
hmm, maybe it's because of variable variable scope ?, it's all get werid after:
```
VERBOSE: GET with 0-byte payload
VERBOSE: received 18861-byte response of content type text/html
VERBOSE: GET with 0-byte payload
VERBOSE: received 18800-byte response of content type text/html
```
I think it's from `inkscape` checkver. it's using `$version` variable and the value is 1.1.1
https://github.com/ScoopInstaller/Extras/blob/master/bucket/inkscape.json#L36

**Comment by LazyGeniusMan** [comment]
> I think it's from `inkscape` checkver. it's using `$version` variable and the value is 1.1.1

Ok so this is the problem. changed it in dummy repo[^1], and then run the github action[^2], it's working[^3].

[^1]: https://github.com/LazyGeniusMan/Scoop-Test/commit/b249b7ba19186cf9da6a86beb0e79a3848fd54fa
[^2]: https://github.com/LazyGeniusMan/Scoop-Test/actions/runs/1645606185
[^3]: https://github.com/LazyGeniusMan/Scoop-Test/commits/master
